less/index.less

```diff
@@ -54,6 +54,35 @@ table {
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
+	background-color: lightblue;
 }
 
+.SecForm {
+	display: flex;
+	background-color: white;
+	align-items: center;
+}
+
+.Main {
+	display: flex;
+	justify-content: center;
+	max-width: 1600px;
```

Comment:
Was this intended so that it will fill the whole screen? If so you could always just use `width: 100%`